src/test/java/dk/kb/storage/integration/DsStorageClientTest.java

```diff
@@ -52,24 +60,44 @@ public class DsStorageClientTest {
 
     private static DsStorageClient remote = null;
     private static String dsStorageDevel=null;  
-
+    
+ 
     @BeforeAll
-    static void setup() {
+    static void setUp() throws Exception{
         try {
             ServiceConfig.initialize("conf/ds-storage-behaviour.yaml","ds-storage-integration-test.yaml"); 
-            dsStorageDevel= ServiceConfig.getConfig().getString("integration.devel.storage"); 
+            dsStorageDevel= ServiceConfig.getConfig().getString("integration.devel.storage");
             remote = new DsStorageClient(dsStorageDevel);
         } catch (IOException e) { 
             e.printStackTrace();
             log.error("Integration yaml 'ds-storage-integration-test.yaml' file most be present. Call 'kb init'"); 
             fail();
-
         }
+        
+        try {            
+            String keyCloakRealmUrl= ServiceConfig.getConfig().getString("integration.devel.keycloak.realmUrl");            
+            String clientId=ServiceConfig.getConfig().getString("integration.devel.keycloak.clientId");
+            String clientSecret=ServiceConfig.getConfig().getString("integration.devel.keycloak.clientSecret");                
+            String token=KeycloakUtil.getKeycloakAccessToken(keyCloakRealmUrl, clientId, clientSecret);           
+            log.info("Retrieved keycloak access token:"+token);            
+            
+            //Mock that we have a JaxRS session with an Oauth token as seen from within a service call.
+            MessageImpl message = new MessageImpl();                            
+            message.put(OAuthConstants.ACCESS_TOKEN_STRING,token);            
+            MockedStatic<JAXRSUtils> mocked = mockStatic(JAXRSUtils.class);           
+            mocked.when(JAXRSUtils::getCurrentMessage).thenReturn(message);
+                                                                         
+        }
+        catch(Exception e) {
+            log.warn("Could not retrieve keycloak access token. Service will be called without Bearer access token");            
+            e.printStackTrace();
+        }                        
```

Comment:
Instead of adding this to each class containing integration tests throughout the services, can we then add a util method which does at least some of the job in kb-util? Maybe only lines 81-94 can be extracted as the first three lines might be service specific.

Reply:
This would require the Mock framework dependencies i KB-util which all projects would then inherit.
The lastest 'mockito-junit-jupiter' will conflict will all older projects using mock framework.

Only good solution would be if we have a ds-util only for ds projects. But unfortunately we have put everything in the general kb-util klass.

Reply:
Good point. Accepting it as is here.